packages/react-wrapper/package.json

```diff
@@ -48,7 +48,7 @@
     "@types/jest": "~27.0.3",
     "@types/react": "~16.9.0",
     "@types/react-dom": "~16.9.0",
-    "cypress": "~8.4.0",
+    "cypress": "~12.0.0",
```

Comment:
there is new version, should we update to it? 12.1.0. Maybe we can use the minor version instead of the patch?